### PR TITLE
Feature/219 faf do you have dfe sign in

### PR DESCRIPTION
**Pull request by d-a-v-e**
## Changes in this PR

* step logic and faf form/schema
* dsi_or_search route, action and view

## Screenshots of UI changes
Screenshot 2022-01-12 at 18 09 50

### CODE REVIEW

**Comment by d-a-v-e** [comment]
(FYI: this is branched off of #760 so currently has some of @PiperS52 's work)

**Review by peterdavidhamilton — CHANGES_REQUESTED** [review]
Can we adopt inheritance here and copy the base classes we have for schema and form that we set up in the `support` namespace first.

File: app/views/fafs/steps/_step_2.html.erb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<span class="govuk-caption-l">
+  step 2
+</span>
+<h1 class="govuk-heading-l">
+  Step 2
+</h1>
+
+<br>
```
Comment (d-a-v-e) [inline app/views/fafs/steps/_step_2.html.erb:8]: (this is a placeholder)

**Comment by EdwinKruglov** [comment]
@d-a-v-e I can see the `Create a new DfE Sign-in account` link in the screenshot, but I don't see it in the code (except for the string in the locale file)?

**Review by peterdavidhamilton — APPROVED** [review]

**Pull request merged**